### ISSUE TITLE
Change auction ban notification in UI

### ISSUE THREAD

**Issue by vohmar** [internetee/auction_center#202]
There is a notice in the auction center Web UI about auction participation restrictions due to overdue invoices. These messages need to be changed

limited ban:
et: "Sul on keelatud osaleda oksjonil järgmis(t)ele domeeninime(de)le: %{domain_name1}, %{domain_name2}."
en: "You are banned from participating in auctions for domain(s) %{domain_name1}, %{domain_name2}."

absolute ban:
et: "Korduvalt tasumata jäetud arvete tõttu on Su õigus osaleda .ee domeenioksjonitel peatatud kuni <date>!"
en: "You are banned from participating in .ee domain auctions due to multiple overdue invoices until <date>! "

**Comment by vohmar** [comment]
https://github.com/internetee/auction_center/pull/200/commits/51ac98740c74c336ea976f711ab9360bb3cb604a
not quite there yet, but solves the domain name variable issue

**Comment by vohmar** [comment]
Good bonus feature here would be to show the message for specific domain auction ban only if the domain is actually at auction or has not been registered / realeased for regular registration yet.

**Comment by maciej-szlosarczyk** [comment]
This is blocked by #198

**Comment by maciej-szlosarczyk** [comment]
Currently, we run the ban query on every page load, this is inefficient but was good enough. What we should do is to run the query on sign in, and stick that information into user cookie or session.

**Comment by vohmar** [comment]
got feedback from troubled user abut current confusing ban notification in the UI.  First priority here is to remove current ban end date in case of limited ban from the notification  as it is technical detail and does not comply with the auction regulations.

**Comment by vohmar** [comment]
littel typo in et message: "Sul on keelatud osaleda oksjonil järgmis(t)ele domeeninime(de)le:: martin54.ee, martin55.ee." double colon before the list of domainnames.

is the ban query run on sign in now or is it still every page load? should I create a separate ticket for this

will create a separate ticket for feature that hides the ban message from UI if the domains user is banned from are not at auctiond. But we will probably need to add count of active offenses (low priority task)

**Comment by maciej-szlosarczyk** [comment]
Typo is fixed now. The query is run on sign in only, so no need for a new ticket. 

Please also check that it works as expected when user logs in with TARA. The shortest route to test that would be to create a ban for a user from administrator UI, those have exactly the same message as the repeated offense ban.
